tests/transformer/models/mixtral_moe/test_modeling_mixtral.py

```diff
@@ -0,0 +1,97 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c)  2023-2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# -----------------------------------------------------------------------------
+
+
+import unittest
+import pytest
+import functools
+import transformers
+from transformers.models.mixtral.modeling_mixtral import MixtralForCausalLM
+
+from QEfficient.utils.constants import Constants
+from QEfficient.utils.device_utils import get_available_device_id, is_multi_qranium_setup_available
+from tests.utils import get_cloud_ai_100_tokens, set_up
+
+
+def get_config():
+    """
+    Function to get config info from transformers.AutoConfig
+    :param: None
+    :return model_config - Dict
+    """
+
+    model_config = {}
+    model_config["model_name"] = "Open-Orca/Mixtral-SlimOrca-8x7B"
+    config = transformers.AutoConfig.from_pretrained(model_config["model_name"])
+    config._attn_implementation = "eager"
+    n_heads = config.num_attention_heads
+    d_head = config.hidden_size // n_heads
+    model_config["model_class"] = MixtralForCausalLM
+    model_config["n_layer"] = config.num_hidden_layers
+    model_config["padding_shape"] = [1, config.num_key_value_heads, Constants.CTX_LEN, d_head]
+
+    return model_config
+
+
+def skip_if_mq_not_enabled(test_method):

```

Comment:
move this to `tests/utils`? can be used by other bigger models we use in future?

Reply:
it is a wrapper function, so keeping it in same class for now